lang/en-gb.xml

```diff
@@ -59,6 +59,8 @@
 		<app_delete>This application and all related data, including saved data, will be deleted.</app_delete>
 		<app_delete_note>Deleting a application may take a while
 depending on its size and your hardware.</app_delete_note>
+		<addcont_delete>Do you want to delete this DLC data?</addcont_delete>
```

Comment:
"DLC data" does not sound _as correct as it could be_. It is unpleasant to say/think about, you should probably keep it as just "DLC".